src/pages/home/getting-started/get-the-sdk.md

```diff
@@ -81,29 +82,9 @@ There are two ways to achieve this:
 
 #### a) Using MobileCore.initialize API (Recommended)
 
-<InlineAlert variant="warning" slots="text" />
-
-This API is available starting from **Android BOM version 3.8.0** and **iOS version 5.4.0**.
-
 The `MobileCore.initialize` API provides a simple way to initialize AEP SDK. It automatically registers solution extensions and enables lifecycle tracking, eliminating the need for manual setup. Refer to the [API documentation](../base/mobile-core/api-reference.md#initialize) for additional configuration options.
 
-<TabsBlock orientation="horizontal" slots="heading, content" repeat="4"/>
```

Comment:
Can you also remove the code related to `add-simplified-initialization` from tabls/get-the-sdk.md?